### DISCUSSION TITLE
Weird Runtime Error at Settings

### DISCUSSION THREAD

**Discussion by kevinamezaga** [solspace/craft-freeform#138] [Questions & Answers] [answered]
So, I just started getting this error on my site when I try to go into the Freeform settings. Never had it before which is interesting. I've tried switching between versions (I'm on the latest: Beta 6) and nothing.
![Screen Shot 2021-05-10 at 19 06 44](https://user-images.githubusercontent.com/25387497/117735241-14c31380-b1c3-11eb-930c-2bd9a3ecb9d8.png)
![Screen Shot 2021-05-10 at 19 08 44](https://user-images.githubusercontent.com/25387497/117735281-27d5e380-b1c3-11eb-9208-3fb8109889f9.png)

**Comment by gustavs-gutmanis** [comment, marked as answer]
That is very strange @kevinamezaga 

```twig
    {{ forms.selectField({
        label: "Freeform Script Insert Type"|t('freeform'),
        name: "settings[scriptInsertType]",
        value: settings.scriptInsertType,
```

From the settings page, it is never called as a function and the property is public.  
Could you take a look at `vendors/solspace/craft-freeform/packages/plugin/src/Models/Settings.php` line `98` and see if there's a `public $scriptInsertType;` line there?

**Reply by kevinamezaga** [reply]
Seems to point to no, @gustavs-gutmanis. I'm going to reinstall from composer.

![Screen Shot 2021-05-11 at 11 13 16](https://user-images.githubusercontent.com/25387497/117840142-e63c4b80-b249-11eb-950a-73d2664963ed.png)

![Screen Shot 2021-05-11 at 11 12 07](https://user-images.githubusercontent.com/25387497/117839985-bdb45180-b249-11eb-8f74-390bd2170fd8.png)
